### ISSUE TITLE
Feature request: support VSCode Insiders edition

### ISSUE THREAD

**Issue by postylem** [mila-iqia/milatools#24]
I'd like to open a ssh connection to a compute node in VSCode Insiders, rather than VSCode (that is, I think as simple as using cli command `code-insiders` rather than `code`). From what I can tell, there isn't currently a way to do this, as running 

```bash
mila code $PATH_ON_CLUSTER
```
will salloc a node $NODE_NAME, and try to connect with it using the command `code -nw --remote ssh-remote+$NODE_NAME $PATH_ON_CLUSTER`.  I'd like to have the option to use `code-insiders ...` instead.

To enable this is I think as simple as modifying `"code",` -> `"code-insiders",` in `milatools/cli/__main__.py` (line 277)
https://github.com/mila-iqia/milatools/blob/b4c52579391f7674d88a8039eddc1bf8a7cca8f1/milatools/cli/__main__.py#L276-L280

But ideally, I'd be able to do this with a different command, like

```bash
mila code-insiders $PATH_ON_CLUSTER
```

Is there a reason why this isn't a good idea?

**Comment by breuleux** [comment]
Although, I feel like you would typically not use both `code` and `code-insiders`, so couldn't you alias the former to the latter?

**Comment by postylem** [comment]
The first thing I tried was to `alias code=‘code-insiders’` but for a reason I don’t understand it doesn’t work; it still runs the command as `code`. I do think though that if one were potentially testing some feature or other in `code-insiders` and get really otherwise using standard `code`, it would be useful to have the switch available as a cli option.

**Comment by breuleux** [comment]
Ah yeah I think it's just because aliases are processed by the shell and `code` is executed with shell=False.

**Comment by postylem** [comment]
Ah, okay.  And I imagine `subprocess.run( ... , shell=True)` is probably not the right solution here.  Would an option to specify the command option or reference an environment variable be the best?

**Comment by breuleux** [comment]
`shell=True` can work, but it requires some care in escaping special characters. I think an option and environment variable are a better solution.

**Comment by breuleux** [comment]
I published version 0.0.11, which adds the flag and environment variable along with another fix.

**Comment by postylem** [comment]
Just to be clear for anyone visiting this thread, as of version 0.0.11, to use VSCode Insiders with milatools, the following will work:

Set environment variable
```bash
export MILATOOLS_CODE_COMMAND=code-insiders
```
Then run 
```bash
mila code .
```
